Bukkit/src/main/java/com/plotsquared/bukkit/listener/PlayerEventListener.java

```diff
@@ -533,7 +533,9 @@ public void onTeleport(PlayerTeleportEvent event) {
                     // to is identical to the plot's home location, and untrusted-visit is true
                     // i.e. untrusted-visit can override deny-teleport
                     // this is acceptable, because otherwise it wouldn't make sense to have both flags set
-                    if (!result && !(plot.getFlag(UntrustedVisitFlag.class) && plot
+                    if (result) {
+                        plotListener.plotEntry(pp, plot);
```

Comment:
Why is this only being added here? Do we not want to call `plotEntry` if deny teleport doesn't allow entry, but the untrusted visit flag does?